risc0/zkvm/src/receipt.rs

```diff
@@ -341,3 +368,12 @@ impl ReceiptMetadata {
         })
     }
 }
+
+/// Compute and return the ImageID of the given `(merkle_root, pc)` pair.
+pub fn compute_image_id(merkle_root: &Digest, pc: u32) -> Digest {
+    use risc0_zkp::core::{digest::DIGEST_WORDS, hash::sha::Sha256};
```

Comment:
I generally think we should avoid in-function imports because it can be hard to track down import scoping when introducing new feature flags.  But its only a small nit.

Reply:
Agreed, but in this particular case, I couldn't figure out how to get it to work because the rest of the file needs `Sha256` but is a different trait with the same name.